src/Components/LoginForm.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+class LoginForm extends Component {
```

Comment:
This component seems very redundant if all it contains is an empty form tag? Where are the props that you're fetching from it on line 30 in the App.js component?

Reply:
Code wasn't pushed up properly since we were pushing to Pia's pull request